src/main/java/de/zalando/aruha/nakadi/exceptions/NoStreamingSlotsAvailable.java

```diff
@@ -0,0 +1,14 @@
+package de.zalando.aruha.nakadi.exceptions;
+
+import javax.ws.rs.core.Response;
+
+public class NoStreamingSlotsAvailable extends NakadiException {
+    public NoStreamingSlotsAvailable(final int totalSlots) {
+        super("No free slots for streaming available. Total slots: " + totalSlots);
+    }
+
+    @Override
+    protected Response.StatusType getStatus() {
+        return Response.Status.BAD_REQUEST;
```

Comment:
Thinking from the point of view of a developer implementing a client for Nakadi subscriptions, how is it possible for me to detect that this scenario is happening? Should I programatically match over the error message? Or is it enough to match on the status code? Is there any other scenarios that could generate a 400 other than this one?